thunderbird/src/main/java/zeta/android/thunderbird/api/devapi/response/pdp/PdpResolutionsResponse.java

```diff
@@ -0,0 +1,79 @@
+package zeta.android.thunderbird.api.devapi.response.pdp;
+
+
+import com.google.gson.annotations.SerializedName;
+
+public class PdpResolutionsResponse {
+
+    @SerializedName("1080X1440Xmini")
+    public String _1080X1440Xmini;
+
+    @SerializedName("48X64")
+    public String _48X64;
+
+    @SerializedName("1080X1440")
+    public String _1080X1440;
+
+    @SerializedName("150X200")
+    public String _150X200;
+
+    @SerializedName("81X108Xmini")
+    public String _81X108Xmini;
+
+    @SerializedName("81X108")
+    public String _81X108;
+
+    @SerializedName("360X480")
+    public String _360X480;
+
+    @SerializedName("180X240")
+    public String _180X240;
+
+    @SerializedName("360X480Xmini")
+    public String _360X480Xmini;
+
+    @SerializedName("180X240Xmini")
+    public String _180X240Xmini;
+
+    @SerializedName("150X200Xmini")
+    public String _150X200Xmini;
+
+    @SerializedName("48X64Xmini")
+    public String _48X64Xmini;
+
+    @SerializedName("125X161")
+    public String _125X161;
+
+    @SerializedName("125X161Xmini")
+    public String _125X161Xmini;
+
+    public PdpResolutionsResponse(String _1080X1440Xmini,
+                                  String _48X64,
+                                  String _1080X1440,
+                                  String _150X200,
+                                  String _81X108Xmini,
+                                  String _81X108,
+                                  String _360X480,
+                                  String _180X240,
+                                  String _360X480Xmini,
+                                  String _180X240Xmini,
+                                  String _150X200Xmini,
+                                  String _48X64Xmini,
+                                  String _125X161,
+                                  String _125X161Xmini) {
+        this._1080X1440Xmini = _1080X1440Xmini;
+        this._48X64 = _48X64;
+        this._1080X1440 = _1080X1440;
+        this._150X200 = _150X200;
+        this._81X108Xmini = _81X108Xmini;
+        this._81X108 = _81X108;
+        this._360X480 = _360X480;
+        this._180X240 = _180X240;
+        this._360X480Xmini = _360X480Xmini;
+        this._180X240Xmini = _180X240Xmini;
+        this._150X200Xmini = _150X200Xmini;
+        this._48X64Xmini = _48X64Xmini;
+        this._125X161 = _125X161;
+        this._125X161Xmini = _125X161Xmini;
```

Comment:
Neat we need to think to give better names for these variables. I know variables stating numbers is not java friendly. Lets deal these kind of things in transformer layer.